package.json

```diff
@@ -40,15 +40,15 @@
     "eslint-plugin-jest": "^22.6.4",
     "eslint-plugin-prettier": "^3.1.0",
     "execa": "^1.0.0",
-    "jest": "^21.0.0 || ^22.0.0 || ^23.0.0 || ^24.0.0 || ^25.1.0",
+    "jest": "^26.0.0",
```

Comment:
This line needs to exactly match the peerdep.

Reply:
Ah sorry thought it was an oversight. Doesn't npm just install the latest?

Reply:
If none is otherwise installed, yes; but in tests, we'll want to install each supported version, and `npm ls` should pass there.